lib/house.rb

```diff
@@ -0,0 +1,39 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price.gsub("$", '').to_i
```

Comment:
nice, I like how you sanitized the price as it comes here in the initialize method